staging/src/k8s.io/apimachinery/pkg/util/wait/wait.go

```diff
@@ -34,7 +34,7 @@ import (
 // The value for timeout should effectively be "forever." Obviously we don't want our tests to truly lock up forever, but 30s
 // is long enough that it is effectively forever for the things that can slow down a run on a heavily contended machine
 // (GC, seeks, etc), but not so long as to make a developer ctrl-c a test run if they do happen to break that test.
-var ForeverTestTimeout = time.Second * 30
+var ForeverTestTimeout = time.Minute * 5
```

Comment:
I don't think this is a good idea. If you need this for a specific test, do it there.

Reply:
Removed, and introduced `customTimeout` only used for large N.